.flowconfig

```diff
@@ -18,4 +18,4 @@ unsafe.enable_getters_and_setters=true
 module.name_mapper='/lib/' -> '/src/'
 
 [version]
-^0.54.0
+^0.55.0
```

Comment:
We still don't want this to specify a particular version of Flow. I.e. leave this file unchanged so that we always use the latest deployed version of Flow.

Reply:
I have different engineering perspective on this:
I understand you want the system flow to be sync with latest release which can tell us to fix flow errors and upgrade. But this time window is really annoying sometimes confusing -- you do not know if the flow errors in Nuclide is because of your feature code change(which you must fix), or just a new flow release(which you can ignore temporarily and fix later). Mentally, you can't trust flow errors in Nuclide anymore. 
I think fixing newer flow errors and upgrading flow to latest version should be an out-of-band task not conflicting with daily feature development. For example, it can be added to on-call's duty list.

Reply:
CI is another thing to consider that needs to be automatically instantly upgraded or it'll start failing any PRs that get submitted in the time window where CI gets upgraded (which is not instantly when Flow is deployed but a different time window).

Reply:
CI won't fail because package.json fixes the flow version it should use. The setting in .flowconfig affects mainly the editing experience.